src/analyzer/plugins/analyzerplugin.h

```diff
@@ -69,8 +70,8 @@ class AnalyzerBeatsPlugin : public AnalyzerPlugin {
     ~AnalyzerBeatsPlugin() override = default;
 
     virtual bool supportsBeatTracking() const = 0;
-    virtual float getBpm() const {
-        return 0.0f;
+    virtual mixxx::Bpm getBpm() const {
+        return mixxx::Bpm();
```

Comment:
How about introducing `static constexpr Bpm::kUndefined = Bpm(kValueUndefined)` in `mixxx::Bpm` for convenience? This would be more obvious than using the default constructor.

A test should verify that default constructed instances equal this constant.

Reply:
I think both is okay. I don't think it's possible to add such a constexpr into the class itself, because it's incomplete at this point. If you know how to do this, please make a suggestion in a separate PR.